scripts/update-abi-registry.js

```diff
@@ -76,15 +76,17 @@ async function main () {
         }
       }
     }
-    target.abi = abiVersion.modules.toString()
 
     const key = [target.runtime, target.target].join('-')
     if (abiVersionSet.has(key)) {
       continue
     }
 
     abiVersionSet.add(key)
-    supportedTargets.unshift(target)
+    supportedTargets.unshift({
+      ...target,
```

Comment:
@MarshallOfSound I'm not entirely sure about the use of the spread operator here. What NodeJS versions are supported by Node ABI? The spread operator seems to be available in Node since v8.6.0 based on this [website](https://node.green/).

Reply:
Using modern syntax in the `scripts` files is fine. There's no Node version requirement for the module itself, so any "modern" syntax changes in, for example, `index.js` would require a new major version bump.

Reply:
Thanks, makes sense! 👍